.github/workflows/create-release-pr.yml

```diff
@@ -52,6 +52,18 @@ jobs:
           echo "$STORIES" >> $GITHUB_ENV
           echo "EOF" >> $GITHUB_ENV
 
+      - name: Compile PR title
+        run: |
+          # Get version from .release-version file
+
+          if [ ! -f .release-version ]; then
+            # read the version from the file
+            VERSION=$(cat .release-version)
+            TITLE="[release] Merge Develop into ${MAIN_BRANCH_NAME^} for ${VERSION}"
+          else
+            TITLE="[release] Merge Develop into ${MAIN_BRANCH_NAME^}"
+          fi
```

Comment:
This looks like it'll be very helpful c:

We could reduce repetition a little and just concatenate the version if it's been changed:

```yml
TITLE="[release] Merge Develop into ${MAIN_BRANCH_NAME^}"

# Check the release version has changed and concat it
if [ -f .release-version ]; then
  VERSION=$(cat .release-version)
  TITLE="${TITLE} for ${VERSION}"
fi
```